### PR TITLE
Update to cope with SDK's new names

### PR DESCRIPTION
**Pull request by drj11**

### CODE REVIEW

**Comment by pwaller** [comment]
Hey :)

I don't understand the commit! These all look like local package changes and nothing to do with the SDK?

File: instance.go [hunk 1]
```diff
@@ -17,13 +17,13 @@ type Instance struct {
 
 func NewInstance(i *ec2.Instance) *Instance {
 	return &Instance{
-		*i.InstanceID, *i.PrivateIPAddress, *i.State.Name,
```
Comment (drj11) [inline instance.go:20]: the `i` here is an ec2.Instance imported from the AWS SDK

File: instance.go [hunk 2]
```diff
@@ -9,7 +9,7 @@ import (
 )
 
 type Instance struct {
-	InstanceID, PrivateIP, State           string
+	InstanceId, PrivateIP, State           string
```
Comment (drj11) [inline instance.go]: I just spotted that I'd also changed the Instance type in this package. I think that was probably a mistake.

**Comment by drj11** [comment]
er, only partly. see my new notes

File: main.go [hunk 3]
```diff
@@ -150,7 +150,7 @@ func main() {
 		fmt.Fprintln(os.Stderr, "[41;1mWarning: agent forwarding not enabled[K[m")
 	}
 
-	client := ec2.New(nil)
+	client := ec2.New(session.New())
```
Comment (pwaller) [inline main.go:153]: Does it not work with a nil session? Curious.

**Comment by pwaller** [comment]
You need to update the import paths from `awslabs` to `aws`.

**Comment by pwaller** [comment]
(It currently doesn't build over here for me)

**Comment by drj11** [comment]
I build this from a completely fresh go install just before I submitted the PR; so, what's up with that? Will investigate. But consider the PR "on hold" for now.

**Comment by drj11** [comment]
I'm building with `go version go1.5.1 linux/amd64`, and it works.

I destroyed my `~/go/src/github.com` directory and running `go install` populates both `aws` and `awslabs` directories. Weird. How does that work? is it following some redirect for `awslabs` and your Go is not?

(still gonna change them, but I'm mystified as to how it possibly works)

**Comment by pwaller** [comment]
Yes, github has redirects for awslabs -> aws (which go in place automagically when you rename or transfer a repository). If you use both then you have a problem, since you're importing them via different names.

File: main.go (hunk 3 above)
Comment (drj11) [inline main.go:153]: It does not work with a `nil` session. _sigh_

**Comment by drj11** [comment]
I `push -f`'d a new branch. Good to go, I think.

**Comment by pwaller** [comment]
Builds for me™.

**Pull request merged**